sample_project/Source/sample_project/Measure/Measure.h

```diff
@@ -80,6 +80,7 @@ class AMeasure : public AActor
 	UFunction* WidgetFunction;
 	double SegmentDistance;
 	FActorSpawnParameters SpawnParam = FActorSpawnParameters();
+	float MarkerHeight = 7000.f;
```

Comment:
Be sure to avoid the `###.f` pattern. Make it more clear and prevents weird linting errors
```suggestion
	float MarkerHeight = 7000.0f;
```